lib/teleterm/daemon/daemon_headless_watcher.go

```diff
@@ -131,11 +133,11 @@ func (s *Service) startHeadlessWatcher(cluster *clusters.Cluster) error {
 					ClientIp:   ha.ClientIpAddress,
 				}
 				if _, err := s.tshdEventsClient.SendPendingHeadlessAuthentication(ctx, req); err != nil {
-					s.cfg.Log.WithError(err).Error("Failed to send pending headless authentication to Electron App.")
+					log.WithError(err).Error("Failed to send pending headless authentication to Electron App.")
 					return
 				}
 			case <-watcher.Done():
-				s.cfg.Log.Debugf("Headless watcher for cluster %v closed with watcher error: %v", cluster.URI.String(), watcher.Error())
+				log.Debugf("Headless watcher closed with watcher error: %v", watcher.Error())
```

Comment:
Same as above with regards to the error. I'm not sure what the best practice is here.